app/src/main/java/com/example/sopt_main/ProfileFragment.kt

```diff
@@ -0,0 +1,71 @@
+package com.example.sopt_main
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import com.bumptech.glide.Glide
+import com.example.sopt_main.databinding.FragmentProfileBinding
+
+
+class ProfileFragment : Fragment() {
+
+    private var _binding : FragmentProfileBinding? = null
+    private val binding get() = _binding!!
+    private lateinit var testViewPagerAdaptor: TestViewPagerAdaptor
+
+
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        _binding = FragmentProfileBinding.inflate(layoutInflater,container,false)
+
+        binding.homeFollowerBtn.isSelected =true
+        binding.homeRepoBtn.isSelected =false
+        transactionFragment()
+        initImage()
+
+        return binding.root
+    }
+
+
+    private fun transactionFragment(){
+
+        val fragment1 =FollowerFragment()
+        val fragment2 =RepositoryFragment()
+
+        childFragmentManager.beginTransaction().add(R.id.profile_fragment,fragment1).commit()
```

Comment:
fragment-ktx 이용해서 아래처럼 보다 프래그먼트 클래스만 참조해서 보다 간결하게 표현이 가능합니다..!
```suggestion
childFragmentManager.commit {
    replace<FollowerFragment>(R.id.home_fragment_cv)
}
```

Reply:
메모리 세이프 하기도 하구용

Reply:
넵!! 하나배워갑니다ㅎㅎ